### PR TITLE
Add project site

### PR DESCRIPTION
**Pull request by danielstreif**
**What**:
- add a new page for a project (AG Selbstbestimmte Bildung)
- link the page to a 'projects' section

**Why**:
- requested by Bianca

image

### CODE REVIEW

File: src/views/agsb.hbs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{{#section "head"}}
+<title>{{{__ "agsb_title"}}} – {{{__ "common_title"}}}</title>
```
Comment (pheeria) [inline src/views/agsb.hbs:2]: question: so we don't have an english version?

**Review by pheeria — APPROVED** [review]

File: src/views/agsb.hbs (hunk 1 above)
Comment (danielstreif) [reply src/views/agsb.hbs:2]: Not at the moment. Bianca just sent me a German version. Shorter texts I usually just translate myself but especially the "Argumente" I don't want to translate freely.

File: src/routes/index.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ const express = require("express");
 
 const router = express.Router();
 
+const agsbRouter = require("./agsb.js");
```
[the remark this answers is not in the document]
Comment (danielstreif) [reply src/routes/index.js:5]: Sure, maybe 'manifesto' for the internal name? Bianca wanted `/agsb` for the url, so I just adopted that.

[the remark this answers is not in the document]
Comment (pheeria) [reply src/routes/index.js:5]: Yes, that is better 🙂

**Review by pheeria — APPROVED** [review]

**Review by jarjan — APPROVED** [review]

File: src/views/agsb.hbs (hunk 1 above)
Comment (danielstreif) [reply src/views/agsb.hbs]: @pheeria I will merge it for now and update it once I have an English version.

**Pull request merged**